AABB_tree/include/CGAL/AABB_polyhedron_triangle_primitive.h

```diff
@@ -154,8 +155,8 @@ namespace CGAL {
     };  // end class AABB_polyhedron_triangle_primitive
 
 
-
 }  // end namespace CGAL
+/// \endcond
```

Comment:
The rules says we **can** **remove** the classes... implicitly: if there is a benefit (for example, easier maintenance). The rules do not say we should remove the documentation for free. And if we remove classes (not the documentation), we usually announce it in the `changes.html`.